src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocNodeImplTest.java

```diff
@@ -21,25 +21,37 @@
 
 import static com.google.common.truth.Truth.assertWithMessage;
 
+import java.util.Arrays;
+
 import org.junit.jupiter.api.Test;
 
+import com.puppycrawl.tools.checkstyle.api.DetailNode;
 import com.puppycrawl.tools.checkstyle.api.JavadocTokenTypes;
 
 public class JavadocNodeImplTest {
 
     @Test
     public void testToString() {
         final JavadocNodeImpl javadocNode = new JavadocNodeImpl();
+        final JavadocNodeImpl child1 = new JavadocNodeImpl();
+        final JavadocNodeImpl child2 = new JavadocNodeImpl();
+        child1.setType(JavadocTokenTypes.CODE_LITERAL);
+        child2.setType(JavadocTokenTypes.CODE_LITERAL);
         javadocNode.setType(JavadocTokenTypes.CODE_LITERAL);
+
+        final DetailNode[] children = {child1, child2};
+
         javadocNode.setLineNumber(1);
         javadocNode.setColumnNumber(2);
+        javadocNode.setChildren(children);
```

Comment:
```suggestion
        final JavadocNodeImpl child1 = new JavadocNodeImpl();
        final JavadocNodeImpl child2 = new JavadocNodeImpl();
        child1.setType(JavadocTokenTypes.CODE_LITERAL);
        child2.setType(JavadocTokenTypes.CODE_LITERAL);
        javadocNode.setChildren({child1, child2});
```
Maybe declare where its actually used, to increase cohesion and reduce coupling.
Ideally, the API would take the type through the constructor, so we could reduce it a lot more.